rpc/dialer.go

```diff
@@ -462,6 +480,8 @@ type perRPCJWTCredentials struct {
 	externalAuthToEntity string
 	creds                Credentials
 	accessToken          string
+	// The static external auth material used against the AuthenticateTo request to obtain final accessToken
```

Comment:
q: it says this is used in with AuthenticateTo, but doing the `With` option causes the externalAuthToEntity field to be cleared in the options. Is that correct?

Reply:
Ah good catch. I think that was from an early copy and paste when i added the With option.  I changed to to remove the authEntity and Credentials{} instead of the authTo. The tests didn't catch that and my e2e test locally add the options added in an order that didn't cause issues.